jekyll/_cci2/orbs-faq.md

```diff
@@ -0,0 +1,116 @@
+
+### Downloading, Integrating, and Testing Orbs
+
+* Question: How do I download, integrate and test Orbs?
+
+* Answer: You can invoke Orbs using the `orbs` stanza in version 2.1 or higher of a CircleCI configuration. For example, if you want to publish an Orb called  `hello-build` in the namespace `circleci` and have a published version `0.0.1`, invoke an orb like the example shown below:
+
+```orbs:
+hello: circleci/hello-build@0.0.1
+```
+
+You may then use the elements of that orb in your configuration by referencing these elements under the key `hello`. For example, if an orb has a job called `hello-build`, you can invoke that job in a workflow like the example shown below.
+
+```workflows:
+  info:
+    jobs:
+      - hello/hello-build
+```
+
+CircleCI will publish a web-based registry viewer in the future so Orbs documentation can be auto-generated. In the meantime, you can always pull the source of Orbs directly. For example, you can run `circleci orb source circleci/hello-build@0.01`
+
+### Build Error When Testing Locally
+
+* Question: Why do I get the following error when testing locally:
+
+```circleci build -c .circleci/jobs.yml --job test
+```
+
+```Error:
+You attempted to run a local build with version 2.1 of configuration.
+```
+
+* Answer: To resolve this error, run `circleci config process` on your configuration and then save that configuration to disk. You then should run `circleci build` (this has now been moved to `circleci local execute`) against the processed configuration.
+
+<!---
+### Environment Variables Not Being Passed at Runtime
+
+Occasionally, when you try to convert a configuration to a 2.0 compatible format, environment variables may not be passed at runtime. For example, if you create a simple configuration in your GitHub repository (for example `https://github.com/yourusername/circle-auto/blob/master/.circleci/echo.yml`) and then call the config using:
+
+```export AUTO_FILE=/Users/yourusername/Desktop/apkpure_app_887.apk
+export AUTO_DIR=.
+circleci build -c .circleci/echo.yml --job test
+```
+
+The config shows:
+
+```#!bin/bash -eo pipefail
+echo file $(AUTO_FILE) dir $(AUTO_DIR)
+file directlySuccess!
+```
+Upon execution, you may see the following response:
+
+.circleci/echo.yml
+
+```version: 2
+jobs:
+  build:
+    docker:
+    - image: circleci/openjdk:8-jdk
+    steps:
+    - checkout
+  test:
+    docker:
+    - image: circleci/openjdk:8-jdk
+    environment:
+    - TERM: dumb
+    steps:
+    - checkout
+    - run:
+        command: "echo file ${AUTO_FILE} dir ${AUTO_DIR}"
+workflows:
+  version: 2
+  workflow:
+    jobs:
+    - build
+    - test```
+
+yourusername/circle-autoAdded by GitHub
+```
+--->
+
+### Standard Process To Access Previously Stored Artifacts
```

Comment:
This feels out of place on an orbs faq. If it stays here, let's please not use the term "plugins" below.

Reply:
agreed, removed